packages/create-block/lib/templates/block/view.js.mustache

```diff
@@ -1,3 +1,22 @@
+/**
+ * Use this file for JavaScript code that you want to run in the front-end 
+ * on posts/pages that contain this block.
+ *
+ * When this file is defined as the value of the `viewScript` property
+ * in `block.json` it will be enqueued when, and only when, viewing the
```

Comment:
Can we simplify this a bit? It's 100% correct, but maybe a tad verbose.
```suggestion
 * in `block.json` it will be enqueued on the front end of the site.
```